packages/mui-system/src/colorManipulator/colorManipulator.test.js

```diff
@@ -77,6 +77,16 @@ describe('utils/colorManipulator', () => {
     it('converts a long alpha hex color to an argb color` ', () => {
       expect(hexToRgb('#111111f8')).to.equal('rgba(17, 17, 17, 0.973)');
     });
+
+    it('warns if the color contains space at the end', () => {
+      let result;
+      expect(() => {
+        result = hexToRgb('#aa0099 ');
```

Comment:
I think it would be great to test a high-level API, like `alpha()` instead. I'm not sure why hexToRgb is public in the first place.